presto-docs/src/main/sphinx/functions/ipaddress.rst

```diff
@@ -0,0 +1,9 @@
+===================
+Ipaddress Functions
+===================
+
+.. function:: ip_prefix(ip_address, prefix_bits) -> Ipprefix
```

Comment:
lower case `ipprefix`

Reply:
I saw functions using both uppercase and lowercase here e.g.
`approx_set(x) -> HyperLogLog`
`merge(qdigest) -> qdigest`
I'll make it lowercase.

Reply:
I think it's using the `StandardTypes` values, or type signature values.